src/writers/blosc.writer.hh

```diff
@@ -0,0 +1,4 @@
+#ifndef ACQUIRE_DRIVER_ZARR_BLOSC_WRITER_HH
+#define ACQUIRE_DRIVER_ZARR_BLOSC_WRITER_HH
+
```

Comment:
Why is this otherwise empty?

Reply:
Good catch, must've been from a `git add .`.